script/Base.s.sol

```diff
@@ -53,10 +54,13 @@ contract BaseScript is Script {
         }
     }
 
-    modifier broadcast() {
-        vm.startBroadcast(broadcaster);
-        _;
-        vm.stopBroadcast();
+    function configureSphinx() public override {
```

Comment:
let's add natspec for this function, referencing sphinx docs/repo

Reply:
> what if we keep the broadcast modifier, and support both types of deployments?

I am with you on this. Until Sphinx becomes really stable, we should keep both the options.